docker-compose.yml

```diff
@@ -3,6 +3,7 @@ services:
     gateway:
         ports:
             - 8080:8080
+            - 8082:8082
```

Comment:
Does this work if you remove the port? It should still be accessible from within the cluster. We don't need to expose it generally.

Reply:
Yes, It does and I have removed `8082` from exposed port.